src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,300 @@
+  `Cross-Origin-Resource-Policy: cross-origin` header if possible. 
+* For cross origin resources that you have no control over:
+    * Use the `crossorigin` attribute in the loading HTML tag if the resource is
+      served with CORS.
+    * Ask the owner of the resource to support either CORS or CORP.
+* For iframes, use CORP and COEP headers as follows: 
+  `Cross-Origin-Resource-Policy: same-origin` and `Cross-Origin-Embedder-Policy: 
+  require-corp`.
+
+{% Aside 'key-term' %}
+It's important that you understand the difference between "same-site" and 
+"same-origin". Learn about the difference at [Understanding same-site and 
+same-origin](/same-site-same-origin).
+{% endAside %}
+
+#### 3. Use the COEP Report-Only HTTP header to assess embedded resources
+
+Before fully enabling COEP, you can do a dry run by using the
+`Cross-Origin-Embedder-Policy-Report-Only` header to examine whether the policy
+actually works. You will receive reports without blocking embedded content.
+Recursively apply this to all documents. For information on the Report-Only HTTP
+header, see [Observe issues using the Reporting
+API](#observe-issues-using-the-reporting-api).
+
+#### 4. Enable COEP
+
+Once you've confirmed that everything works, and that all resources can be 
+successfully loaded, apply the `Cross-Origin-Embedder-Policy: require-corp` HTTP 
+header to all documents including those that are embedded via iframes.
+
+{% Aside 'codelab' %}
+[See the impact of different COEP / CORP 
+parameters](https://first-party-test.glitch.me/coep).
+{% endAside %}
+
+### Determine whether isolation succeeded with `self.crossOriginIsolated`
+
+The `self.crossOriginIsolated` property returns `true` when the web page is in a 
+cross-origin isolated state and all resources and windows are isolated within 
+the same browsing context group. You can use this API to determine whether you 
+have successfully isolated the browsing context group and gained access to 
+powerful features like `performance.measureMemory()`.
+
+{% Aside 'caution' %}
+The [`self.crossOriginIsolated`](https://www.chromestatus.com/feature/5953286387531776)
+property is still under development and not available yet in Chrome as of version 83.
+{% endAside %}
+
+### Debug issues using Chrome DevTools
+
+For resources that are rendered on the screen such as images, it's fairly easy 
+to detect COEP issues because the request will be blocked and the page will
+indicate a missing image. However, for resources that don't 
+necessarily have a visual impact, such as scripts or styles, COEP issues might 
+go unnoticed. For those, use the DevTools Network panel. If 
+there's an issue with COEP, you should see 
+`(blocked:NotSameOriginAfterDefaultedToSameOriginByCoep)` in the **Status**
+column.  
+![Debugging COEP in DevTools 1](devtools1.png)
+
+You can then click the entry to see more details.   
+![Debugging COEP in DevTools 2](devtools2.png)
```

Comment:
```suggestion
column.

<figure class="w-figure">
  <img class="w-screenshot-filled" src="devtools1.png" alt="COEP issues in the Status column of the Network panel.">
</figure>

You can then click the entry to see more details.

<figure class="w-figure">
  <img class="w-screenshot-filled" src="devtools2.png" alt="Details of the COEP issue are shown in the Headers tab after clicking a network resource in the Network panel.">
</figure>
```

@agektmr & @jpmedley please make the image `alt` descriptions more descriptive next time. My general approach is to just describe the exact UI that's shown in the screenshot in detail. The DevTools UI is starting to become accessible so there might legitimately be interest from people with low-vision on how to use DevTools to debug this problem.